hooks/post/30-upgrade.rb

```diff
@@ -1,3 +1,7 @@
+require 'fileutils'
+
+STEP_DIRECTORY='/etc/foreman-installer/hooks/post/'
```

Comment:
I didn't see anything in @kafo.config.app that was useful.  Just references to:

/etc/foreman-installer/scenarios.d/katello-answers.yaml
and
/etc/foreman-installer/scenarios.d

i could parse /etc/foreman-installer/scenarios.d and go up one or something, but that might be just as risky as hardcoding.